.github/workflows/ci.yaml

```diff
@@ -300,13 +300,13 @@ jobs:
           args: --target=thumbv6m-none-eabi -p printerdemo_mcu --no-default-features --features=mcu-board-support/pico-st7789
 
   docs:
-    uses: slint-ui/slint/.github/workflows/build_docs.yaml@master
+    uses: ./.github/workflows/build_docs.yaml
```

Comment:
It needs to be written *exactly* like this: Github uses the `./` to decide whether or not to use a file from the repository. Anything else will be retrieved online.

I used to try to include `.github/workflows/foo` and that failed :-/

Reply:
I recall digging deep into this and finding that this just wasn’t supported at the time, despite the ambiguous documentation.

Note: we won’t find out if it works in this patch as of course this will only be active after landing. So the CI results are likely misleading.

I suggest to try this in a little personal test repo first, with two branches. If it works there, then make a change set separate from this one to implement this for Slint. Land & Test it and then rebase this refactoring.

Reply:
I marked this as unresolved since I was tagged and my response would not be seen. I suggest to resolve conversations when concluded by everyone or if the matter is trivial.

Reply:
>Note: we won’t find out if it works in this patch as of course this will only be active after landing. So the CI results are likely misleading.

I think we'll find out in this CI run since this changes ci.yaml in this branch

Reply:
@tronical: The change should be visible right away: The ci.yaml file is in the expected place and the CI runner should error out if it can not find the included file.

The `wasm/wasm` job is also running ./.github/actions/setup-rust as you can see in the details of that job, which is triggered in this branch but not in main.

Reply:
Good point:)